### ISSUE TITLE
Saved configurations are not loaded right on v0.8.9 (and 0.8.7, 0.8.8)

### ISSUE THREAD

**Issue by hellishvictor** [pedrolcl/VMPK#7]
Hi, I've saved some configurations where the "Controllers" are unchecked and the "Extra Controls" (Sustain, Sostenuto and Soft) are removed, but they're loaded again when one of those saved conf are loaded. Also, the first conf one have some custom "Extra Controllers" but they are not created neither. It's very frustrating. Here are one of them:

```
[Window]
Geometry=@ByteArray(\x1\xd9\xd0\xcb\0\x3\0\0\0\0\0\0\0\0\0\0\0\0\x3\xe3\0\0\0\xe9\0\0\0\b\0\0\0\x1e\0\0\x3\xdb\0\0\0\xe1\0\0\0\0\0\0\0\0\x5V\0\0\0\b\0\0\0\x1e\0\0\x3\xdb\0\0\0\xe1)
State=@ByteArray(\0\0\0\xff\0\0\0\0\xfd\0\0\0\0\0\0\x3\xd4\0\0\0J\0\0\0\x4\0\0\0\x4\0\0\0\b\0\0\0\b\xfc\0\0\0\x2\0\0\0\x2\0\0\0\x2\0\0\0\x18\0t\0o\0o\0l\0\x42\0\x61\0r\0N\0o\0t\0\x65\0s\x1\0\0\0\0\xff\xff\xff\xff\0\0\0\0\0\0\0\0\0\0\0\x1e\0t\0o\0o\0l\0\x42\0\x61\0r\0P\0r\0o\0g\0r\0\x61\0m\0s\x1\0\0\x1\xdc\xff\xff\xff\xff\0\0\0\0\0\0\0\0\0\0\0\x2\0\0\0\x3\0\0\0$\0t\0o\0o\0l\0\x42\0\x61\0r\0\x43\0o\0n\0t\0r\0o\0l\0l\0\x65\0r\0s\x1\0\0\0\0\xff\xff\xff\xff\0\0\0\0\0\0\0\0\0\0\0\x18\0t\0o\0o\0l\0\x42\0\x61\0r\0\x45\0x\0t\0r\0\x61\x1\0\0\x1w\xff\xff\xff\xff\0\0\0\0\0\0\0\0\0\0\0\x1a\0t\0o\0o\0l\0\x42\0\x61\0r\0\x42\0\x65\0n\0\x64\0\x65\0r\x1\0\0\x2\x93\xff\xff\xff\xff\0\0\0\0\0\0\0\0)

[DrumstickRT]
PublicNameIN=VMPK Input
PublicNameOUT=VMPK Output

[Connections]
InEnabled=false
ThruEnabled=false
OmniEnabled=false
AdvancedEnabled=false
InputDriver=
OutputDriver=
InPort=
OutPort=01. Internal MIDI

[Preferences]
Channel=14
Velocity=127
BaseOctave=1
Transpose=0
NumKeys=88
InstrumentsDefinition=:/vpiano/gmgsxg.ins
InstrumentName=General MIDI
AlwaysOnTop=false
ShowStatusBar=true
VelocityColor=true
EnforceChannelState=false
EnableKeyboardInput=true
EnableMouseInput=true
EnableTouchInput=false
DrumsChannel=-1
StartingKey=9
CurrentPalette=0
ShowColorScale=false
Language=es_ES
StickyWindowSnapping=true
ForcedDarkMode=false
QtStyle=fusion

[Keyboard]
RawKeyboardMode=false
MapFile=default
RawMapFile=default

[TextSettings]
namesFont="Helvetica,50"
namesOrientation=0
namesVisibility=0
namesAlteration=0
namesOctave=1
octaveSubscript=true

```
I'm using the latest v0.8.9 on Win 7 x64 SP1.
Cheers.

**Comment by pedrolcl** [comment]
This is probably a bug in restoring settings that exists in versions 0.8.9, 0.8.8 and 0.8.7, not only in Windows, but all operating systems. It works in [v0.8.6](https://sourceforge.net/projects/vmpk/files/vmpk/0.8.6/)

Sourceforge [ticket #84](https://sourceforge.net/p/vmpk/bugs/84/)

**Comment by pedrolcl** [comment]
Fixed in commit 97dcb61
